Makefile.global

```diff
@@ -113,6 +113,7 @@ clean:
 	find . -name \*.gcno -o -name \*.gcda | xargs rm -f
 	find . -name \*.lo -o -name \*.o | xargs rm -f
 	find . -name \*.la -o -name \*.a | xargs rm -f
+	find . -name \*.diff -o -name \*.out -o -name \*.log | xargs rm -f
```

Comment:
This will also catch some unrelated stuff, like `config.log`. This probably needs to be restricted to `tests/` directories or something.

Reply:
I've remove this for now, keeping only `*.diff` and `*.out`. I'll figure out a better clean scripts for our `tests` so we can even include `*.php` and `*.exp` as well :)